src/hidimstat/permutation_test.py

```diff
@@ -1,25 +1,16 @@
 import numpy as np
 from joblib import Parallel, delayed
 from sklearn.base import clone
-from sklearn.model_selection import GridSearchCV
-from sklearn.pipeline import Pipeline
-from sklearn.svm import LinearSVR
-from sklearn.utils import _safe_indexing
-
-from hidimstat.stat_tools import pval_from_two_sided_pval_and_sign
-
-
-def permutation_test_cv(
-    X,
-    y,
-    n_permutations=1000,
-    C=None,
-    Cs=np.logspace(-7, 1, 9),
```

Comment:
Sorry, I don't understand why you remove the possibility to learn the optimal C by CV. You don't want to use an SVR (or any other ML model)  without CV.